### PR TITLE
Added a financialYears method to the range class

### PR DESCRIPTION
**Pull request by regatta-ajd**
Also copied some of the current `years` iterator tests

### CODE REVIEW

File: src/Iterator/FinancialYears.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        $this->start = FinancialYear::fromInt($start->getFinancialYear())->getStart()->timestamp();
+        $this->end = FinancialYear::fromInt($end->getFinancialYear())->getStart()->timestamp();
+        $this->rewind();
+    }
+
+
+    /**
+     * Get the current iterator value.
+     *
+     * @return Year
```
Comment (duncan3dc) [inline src/Iterator/FinancialYears.php]: `FinancialYear`

File: src/Range.php [hunk 2]
```diff
@@ -98,4 +98,14 @@ public function seasons()
     {
         return new Iterator\Seasons($this->start, $this->end);
     }
+
+    /**
+     * Get an iterator for the financial years covered by this date range.
+     *
+     * @return Iterator\Years
```
Comment (duncan3dc) [inline src/Range.php]: `Iterator\FinancialYears`

**Comment by regatta-ajd** [comment]
@duncan3dc 
Corrected the comments

**Comment by duncan3dc** [comment]
Thanks :+1:

**Comment by regatta-ajd** [comment]
@duncan3dc 
Cheers

**Pull request merged**